tests/reports.test.js

```diff
@@ -12,21 +12,90 @@ var nowData = {
 };
 
 var someData = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['someData' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/963452030/issues/source?bid=2452379&fileBranchId=2859059#l14)